Makefile

```diff
@@ -15,7 +17,14 @@ vagrant-up:
 aws: aws-provision ansible-galaxy site
 
 aws-provision: 
-	ansible-playbook -i localhost, aws-provision.yml -v --ask-vault-pass
+	ansible-playbook -i localhost, aws-provision.yml -v
 
 site: 
-	ansible-playbook -i ec2.py site.yml -v --ask-vault-pass
+	ansible-playbook -i ec2.py site.yml -v
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys $(var);)
```

Comment:
Yeah, for the refactoring (wrapping lines). But I don't think we should merge this with different behaviour (keys imported without prompt) than the other project.